rclc/package.xml

```diff
@@ -4,7 +4,9 @@
   <name>rclc</name>
   <version>0.0.0</version>
   <description>The ROS client library in C.</description>
-  <maintainer email="william@osrfoundation.org">William Woodall</maintainer>
+  <maintainer email="jan.staschulat@de.bosch.com">Jan Staschulat</maintainer>
+  <maintainer email="julianbermudez@eprosima.com">Julián Bermúdez Ortega</maintainer>
+  <author> Jan Staschulat</author>
```

Comment:
Unnecessary white space before the name.

Reply:
updated package.xml